### ISSUE TITLE
Add `aria`prop to `tabs` component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3789]
Scope
---
Currently there is no possibility to enhance meaning of the `tablist` of the `tabs` component. To provide more individual information to assistive technology users, an `aria-label` should be added to the `tablist` trough the `aria` prop.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Share any further information that is relevant for processing.

Acceptance criteria
---
- [ ] An `aria-label` can be set to the `tablist` item

Subtasks
---
- [ ] Task
